spec/lib/qualifications_api/teacher_spec.rb

```diff
@@ -80,6 +80,38 @@
       )
     end
 
+    it "returns human readable ITT result values" do
+      {
+        "DeferredForSkillsTests" => "Deferred for skills tests",
+        "ApplicationReceived" => "Application received",
+        "ApplicationUnsuccessful" => "Application unsuccessful",
+        "InTraining" => "In training",
+        "NoResultSubmitted" => "No result submitted",
+        "UnderAssessment" => "Under assessment",
+      }.each do |k, v|
+        api_data["initialTeacherTraining"][0]["result"] = k
+        teacher = described_class.new(api_data)
+        qualifications = teacher.qualifications
+        expect(qualifications.find { |q| q.type == :itt }.details.result).to eq(v)
+      end
```

Comment:
All these examples appear to be the same. None of them could fail individually right?

If so, then I'd use a single example to avoid confusing things.

```suggestion
       api_data["initialTeacherTraining"][0]["result"] = "DeferredForSkillsTests"
       teacher = described_class.new(api_data)
       qualifications = teacher.qualifications
       expect(qualifications.find { |q| q.type == :itt }.details.result).to eq("Deferred for skills tests")
```